qcodes/tests/test_abstract_instrument.py

```diff
@@ -78,7 +78,7 @@ def __init__(self, name: str):
         super().__init__(name)
 
     def __post_init__(self, *args: Any, **kwargs: Any) -> None:
-        super().__post_init__()
+        super().__post_init__()  # type: ignore[misc]
         self.call_count += 1
```

Comment:
since post init does not exist, shall we remove it from here as well?

Reply:
This class is only used in the skipped tests. Removing it from here would make the test harder to reestablish

Reply:
assuming that it's a similar thing that will be re-established :)

Reply:
ok, fair